web-api/migration-terraform/main/lambdas/migrations/0004-standing-pretrial-order-signatures.js

```diff
@@ -0,0 +1,89 @@
+const createApplicationContext = require('../../../../src/applicationContext');
+const {
+  CHIEF_JUDGE,
+  SYSTEM_GENERATED_DOCUMENT_TYPES,
+} = require('../../../../../shared/src/business/entities/EntityConstants');
+const {
+  DocketEntry,
+} = require('../../../../../shared/src/business/entities/DocketEntry');
+
+const applicationContext = createApplicationContext({});
+
+const migrateItems = async (items, documentClient) => {
+  const itemsAfter = [];
+  for (const item of items) {
+    if (item.pk.includes('case|') && item.sk.includes('docket-entry|')) {
+      if (
+        item.eventCode ===
+          SYSTEM_GENERATED_DOCUMENT_TYPES.standingPretrialOrder.eventCode ||
+        item.eventCode === 'SPOS'
+      ) {
+        if (!item.signedAt || !item.signedJudgeName || !item.signedByUserId) {
+          const docketNumber = item.pk.split('|')[1];
+
+          const caseRecord = await documentClient
+            .get({
+              Key: {
+                pk: `case|${docketNumber}`,
+                sk: `case|${docketNumber}`,
+              },
+              TableName: process.env.SOURCE_TABLE,
+            })
+            .promise()
+            .then(res => {
+              return res.Item;
+            });
+
+          if (!caseRecord.trialSessionId) {
+            item.signedAt = applicationContext
+              .getUtilities()
+              .createISODateString();
+            item.signedJudgeName = CHIEF_JUDGE;
+            // Hard code signedByUserId as it's not really used for anything but
+            // is required when signedJudgeName has a value
+            item.signedByUserId = '60f2059d-3831-4ed1-b93b-8c8beec478a4';
```

Comment:
We don't use this id for anything, so we didn't think it was necessary. If you disagree we can definitely make that change. This will really only affect old data on dev, though.

Reply:
Right on. I don't think it needs to change in this PR or in this migration script. I just want to be sure we don't make assumptions about `signedByUserId`, as this comment and update suggests that it's basically required junk data.  And so wanting to be sure there's a backlog item to make it work for us (or get removed) in the future.

Reply:
That is a good point! I will backlog looking into the requirement for `signedByUserId` and if it is even necessary.

Reply:
I looked some more, and `signedByUserId` is probably required because it is the user id of the _user_ who signed the document, not the judge. So I think it is probably useful when an actual user signs a document to know who signed it. But it will just be junk data when coming from a migration, because the migration is the one signing it. 😄

Reply:
Thanks!